app/src/lib/stores/pull-request-store.ts

```diff
@@ -318,7 +328,9 @@ export class PullRequestStore {
         return fatalError('PR cannot have a null base repo')
       }
 
-      const baseGitHubRepo = await upsertRepo(endpoint, pr.base.repo)
+      const baseGitHubRepo = repository.fork
+        ? repository
```

Comment:
I'm not sure I understand what this change is doing, could you walk me through it?